README.md

```diff
@@ -76,6 +76,11 @@ Additionally, check out our [blog ](https://blog.kubecost.com/blog/cost-monitori
 
 ## Staying in the loop
 
-You can stay up to date with Kubecost by following releases on [GitHub](https://github.com/kubecost/cost-analyzer-helm-chart/releases).
+You can stay up-to-date with Kubecost by following releases on [GitHub](https://github.com/kubecost/cost-analyzer-helm-chart/releases) and the official [Kubecost blog](https://blog.kubecost.com/).
+
+Follow us on social media:
```

Comment:
We have a Contact Us page at the top of the docs site for this content

Reply:
Thanks for that and sorry I missed it!